### ISSUE TITLE
[SA] Health bar is shifted to the right relative to armor and breath.

### ISSUE THREAD

**Issue by ghost** [CookiePLMonster/SilentPatch#137]
Health bar is shifted to the right relative to armor and breath.

![bug](https://github.com/user-attachments/assets/dbed6228-5edc-4b50-bc95-74e0f67e5848)

- 1.0 US Compact
- Silent's ASI Loader 1.3
- SilentPatch 1.1 Build 33.1
- Widescreen Fix (ThirteenAG) May 12, 2019

**Comment by SilentMRG** [comment]
Armor and breath bars do not increase over time. Yes, the health bar increases as the game progresses, based on the player's actions (eating, working out, increasing stamina), so I think that's how it is, I'm assuming you're at the beginning of the game.

**Comment by CookiePLMonster** [comment]
> * Widescreen Fix (ThirteenAG) May 12, 2019

Update your widescreen fix and check again. And check the same without the widescreen fix too.

**Comment by ghost** [comment]
Nothing changed with latest developer build of Widescreen Fix (Dec 13, 2024).
Without widescreen fix, the HUD looks like this:

![bug2](https://github.com/user-attachments/assets/8eabbcb1-a7db-4797-81c4-c6147d703187)
1920x1080x32
Much better, but there is a difference of one pixel. The fist icon has become square.

![bug3](https://github.com/user-attachments/assets/f82fe4e0-8f52-42a9-921f-fad4dcab730f)
800x600x32
The problem persists on the standard game resolution.

**Comment by CookiePLMonster** [comment]
OK, that looks good enough to me. Just a note though - the "standard" resolution for the game is 640x480, not 800x600, and I suspect there it'll look just fine. It's likely only very slightly different and the differences surface when scaling up, like on your 800x600 screenshot.

**Comment by ghost** [comment]
When I deleted ASI loader and gta_sa.set, the game started at 800x600 resolution.
640x480 was the default resolution for III and VC as far as I remember.
In SA you can select 640x480, but even here the health bar is displayed shifted.

I think this problem is similar to the first save file when his empty.

![bug4](https://github.com/user-attachments/assets/bd2ba0d5-9131-41c3-97ba-1065c16229d6)

**Comment by CookiePLMonster** [comment]
> When I deleted ASI loader and gta_sa.set, the game started at 800x600 resolution.

Yes but the UI elements are laid out for 640x480 and scaled up, so 640x480 should be used as a reference resolution for UI scaling in San Andreas too.
